sdv/multi_table/hma.py

```diff
@@ -44,6 +44,88 @@ def __init__(self, metadata, locales=None, verbose=True):
             self._table_synthesizers,
             self._table_sizes)
 
+    def _num_extended_columns(self, table_name, parent_name, columns_per_table):
+        """Get the number of columns that will be generated for table_name.
+
+        A table generates:
+            - 1 num_rows column for each for each foreign key with a specific parent
+            - n*(n-1)/2 correlation columns for each data column
+            - 4 parameters columns for each data column, with:
+                - 1 column for parameter a
+                - 1 column for parameter b
+                - 1 column for parameter scale
+                - 1 column for parameter loc
```

Comment:
are these the parameters for the default distribution? Maybe we should see what distribution is passed by the user as the default since that happens before fitting, and base the number of parameters off of that